pages/glossary/required-input-field.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Required Input Field
+key: required-input-field
+---
+
+Any [input element](https://www.w3.org/TR/html52/sec-forms.html#the-input-element) that, if left unfilled, prevents the successful submission or processing of the input.
```

Comment:
Asides, should we also highlight the fact that, this can be any input element which as `required` attribute?